src/cascadia/TerminalSettingsEditor/Profiles_Terminal.xaml

```diff
@@ -70,6 +70,15 @@
                 <TextBox Style="{StaticResource TextBoxSettingStyle}"
                          Text="{x:Bind Profile.AnswerbackMessage, Mode=TwoWay}" />
             </local:SettingContainer>
+
+            <!--  Allow VT Clipboard Writing  -->
+            <local:SettingContainer x:Uid="Profile_AllowVtClipboardWrite"
+                                    ClearSettingValue="{x:Bind Profile.ClearAllowVtClipboardWrite}"
+                                    HasSettingValue="{x:Bind Profile.HasAllowVtClipboardWrite, Mode=OneWay}"
+                                    SettingOverrideSource="{x:Bind Profile.AllowVtClipboardWriteOverrideSource, Mode=OneWay}">
+                <ToggleSwitch IsOn="{x:Bind Profile.AllowVtClipboardWrite, Mode=TwoWay}"
+                              Style="{StaticResource ToggleSwitchInExpanderStyle}" />
+            </local:SettingContainer>

```

Comment:
nit: I think the page would look nicer if this was below "Allow VT Checksum Report"

Reply:
![{35DEB2DB-8315-4923-A666-2E178883A34C}](https://github.com/user-attachments/assets/576692e1-b369-4e8f-a153-bfd12558d714)

Reply:
ah thanks, yes